src/components/SetCreditorOrDebitor.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormControl, InputLabel, MenuItem, Select, SelectChangeEvent } from "@mui/material";
+import { User } from "../types/types";
+import UserElement from "./UserElement";
+
+type SetCreditorOrDebitorType = {
+    label: string,
+    onChange: ((event: SelectChangeEvent<any>, child: React.ReactNode) => void) | undefined,
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/SE-DueTo/dueto-frontend/pullRequest?prid=9512455)
```suggestion
    onChange: ((event: SelectChangeEvent<unknown>, child: React.ReactNode) => void) | undefined,
```